src/util/string_utils.cpp

```diff
@@ -87,7 +106,11 @@ void split_string(
   std::string &right,
```

Comment:
Cheeky of me, but as you are touching this function, do you fancy adding some Doxygen? ;-)